gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannel.java

```diff
@@ -192,16 +184,20 @@ public Object call() throws IOException {
       // the writer at the other end will not hang indefinitely.
       // Send the initial StartResumableWrite request to get an uploadId.
       uploadId = startResumableUpload();
-      Retryer<Object> retryer = getRetryer();
       try (InputStream ignore = pipeSource) {
-        return retryer.call(this::doResumableUpload);
-      } catch (ExecutionException | RetryException e) {
+        return ResilientOperation.retry(
+            this::doResumableUpload,
+            backOffFactory.newBackOff(),
```

Comment:
how many times this retry going to be? I didn't see it's specified for this newBackOff?

Reply:
Right now it has 10 retries by default:
https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/9e0c362c20bc464303ddd23d49b32f7c34724924/gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java#L162-L164